### PR TITLE
PR for Edit and Archive 

### PR DESCRIPTION
**Pull request by manismk**
- Added Edit functionality
- Added Archive and unarchive functionality
- [Link](https://deploy-preview-8--key-notes.netlify.app/)

### CODE REVIEW

File: src/utils/updateNotes.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export const updateNotes = (notesData, uid) => {
+  try {
+    db.collection(`users/${uid}/notes`).doc(notesData.id).update({
+      title: notesData.title,
+      enteredNotes: notesData.enteredNotes,
+      isPinned: notesData.isPinned,
+      color: notesData.color,
+    });
+  } catch (e) {
```
Comment (JowelTisso) [inline src/utils/updateNotes.js:11]: Suggestion: you can destructure notesData in place as well, so you dont have to repeat notesData

**Review by JowelTisso — APPROVED** [review]
Good use of template literals, Rest looks good.

**Review by muskaanshah — APPROVED** [review]
Code looks good, good naming convention.
Suggestion on the home page - add some padding to the button (the start note taking journey one), this odesn't look like your CTA because of the size

File: src/utils/updateNotes.js (hunk 1 above)
Comment (JowelTisso) [reply src/utils/updateNotes.js:11]: Also add favicon

Comment (manismk) [reply src/utils/updateNotes.js:11]: will do that

**Pull request merged**